.github/actions/release-image/action.yml

```diff
@@ -32,19 +32,19 @@ runs:
 
     # If publishing to GCR, setup OIDC->SA auth
     - id: gcrauth1
-      if: startsWith(inputs.apkoBaseTag, 'gcr.io/')
+      if: startsWith(inputs.apkoBaseTag, 'gcr.io/') || startsWith(inputs.apkoBaseTag, 'pkg.dev/')
```

Comment:
Is `startsWith` sufficient? pkg.dev repos are horrible monstrosities like `us-central1-docker.pkg.dev/blah/blah`

Reply:
good call. resolved!